docs/integrations/prefect-dbt/index.md

```diff
@@ -1,22 +1,54 @@
 # prefect-dbt
 
-With prefect-dbt, you can trigger and observe dbt Cloud jobs, execute dbt Core CLI commands, and incorporate other tools, such as Snowflake, into your dbt runs.
+With prefect-dbt you can trigger and observe dbt Cloud jobs, execute dbt Core CLI commands, and incorporate other tools, such as Snowflake, into your dbt runs.
 Prefect provides a global view of the state of your workflows and allows you to take action based on state changes.
 
-## Getting started 
+## Getting started
 
-1. [Install prefect-dbt](#installation) 
-1. [Register newly installed blocks types](#registering-block-types)
+### Prerequisites
 
-Explore the examples below to learn how to use Prefect with dbt.
+- [Prefect installed](/getting-started/installation/).
+- A [dbt Cloud account](https://cloud.getdbt.com/signup) if using dbt Cloud.
 
-### Integrate dbt Cloud jobs with Prefect flows
+### Install prefect-dbt
 
-If you have an existing dbt Cloud job, you can use the pre-built flow `run_dbt_cloud_job` to trigger a job run and wait until the job run is finished.
+Install `prefect-dbt` for use with dbt Cloud:
+
+<div class = "terminal">
+```bash
+pip install -U prefect-dbt
+```
+</div>
+
+Install with additional functionality for dbt Core (CLI):
+
+<div class = "terminal">
+```bash
+pip install -U "prefect-dbt[cli]"
```

Comment:
I think this section can be omitted.  `dbt_core` is not an optional dependency, currently.